src/pykoala/corrections/flux_calibration.py

```diff
@@ -26,21 +26,47 @@
 
 class FluxCalibration(CorrectionBase):
     """
-    This module contains the methods for extracting and applying an absolute flux calibration.
-
-    Description
-    -----------
-
+    A class to handle the extraction and application of absolute flux calibration.
 
     Attributes
     ----------
+    name : str
+        The name of the Correction.
+    verbose : bool
+        If True, prints additional information during execution.
+    calib_spectra : None or array-like
+        The calibration spectra data.
+    calib_wave : None or array-like
+        The calibration wavelength data.
+    response : None or array-like
+        The response data.
+    resp_wave : None or array-like
+        The response wavelength data.
+    response_units : float
+        Units of the response function, default is 1e16 (erg/s/cm^2/AA).
 
     Methods
     -------
-
-    Example
-    -------
+    interpolate_model(wavelength, update=True)
+        Interpolates the spectral response to the input wavelength array.
+    auto(data, calib_stars, extract_args=None, response_params=None, save=None, fnames=None, combine=False)
+        Performs automatic calibration using a set of standard stars.
+    master_flux_auto(flux_cal_results)
+        Computes a master response function from results returned by the auto method.
+    extract_stellar_flux(data_container, wave_range=None, wave_window=None, profile=cumulative_1d_moffat, bounds='auto', growth_r=np.arange(0, 10, 0.5), plot=False, **fitter_args)
+        Extracts the stellar flux from a DataContainer object.
+    list_available_stars(verbose=True)
+        Lists all available spectrophotometric standard star files.
+    read_calibration_star(name=None, path=None, flux_units=None)
+        Reads the spectra of a calibration star from a file.
+    get_response_curve(wave, obs_spectra, ref_spectra, pol_deg=None, spline=False, spline_args={}, gauss_smooth_sigma=None, plot=False, mask_absorption=True)
+        Computes the response curve from observed and reference spectra.
+    apply(data_container, response=None, response_units=None)
+        Applies the spectral response function to a DataContainer object.
+    save_response(fname, response, wavelength, units=None)
+        Saves the response function to a file.
```

Comment:
Isn't this created automatically by Sphinx?

Reply:
Yes, I thought Sphinx would be smart enough to avoid duplicating documentation. The reason why I included that (or better said ChatGPT did) is because I find very useful to have an overview of the class methods when doing something like `FluxCalibration?` on a ipython console. I removed it.